### PR TITLE
Added jenkins, build and test cmd files and updated path of bin folder

### PR DESCRIPTION
**Pull request by yyang-convey**
The file in "X:\__Dev\__Source\csharp-script\dependencies\nunit.framework.dll" is an invalid file so then changed it to the version 2.6.2 in the program files folder.

### CODE REVIEW

File: csharp-code-evaluator-ut/csharp-script-ut.csproj [hunk 1]
```diff
@@ -36,7 +36,7 @@
   <ItemGroup>
     <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\dependencies\nunit.framework.dll</HintPath>
+      <HintPath>C:\Program Files (x86)\NUnit 2.6.2\bin\framework\nunit.framework.dll</HintPath>
```
Comment (jsbattig) [inline csharp-code-evaluator-ut/csharp-script-ut.csproj]: @yyang-convey , why do you change to use hard coded version in c:\Program Files\.... ?
The repo is already configured to use version included in the project. You need to have git-lfs in your computer, and you will get the specific version of nunit.
Actually, we could deploy nunit-console exactly in the same way

Comment (yyang-convey) [inline csharp-code-evaluator-ut/csharp-script-ut.csproj]: @jsbattig I re-clone the repo and I was able to pull the dll.  I pushed new changes that doesn't have this code change, but GIT is not removing it from here.

Comment (yyang-convey) [inline csharp-code-evaluator-ut/csharp-script-ut.csproj]: I did a force update, so it has two new commits.

**Comment by yyang-convey** [comment]
@jsbattig its ready I have removed the sub-directory of SampleApp

**Pull request merged**